src/style.css

```
@@ -0,0 +1,78 @@
+html {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+header {
+  align-items: center;
+  margin: 0 50px;
+  display: flex;
+  justify-content: space-between;

```

внутрішні відступи це падінги. Також зберігай правильний порядок

```suggestion
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 50px;
```